en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+        If you do not create a new project, you can create the REST API for the default project.
+    
+    1. On the Home page of the Choreo Console, click **+ Create Project**.
+
+    2. Enter a unique name and a description for the project.
+
+    3. Click **Create**.
+
+## Step 1: Create and publish a REST API
+
+In this step, you are playing the role of John, the API developer. You will create and publish the REST API that the application developed by Sam needs to consume.
+
+### Step 1.1: Create the REST API
+
+Let's create your first REST API.
+
+1. On the Home page, click on the required project.
+
+2. If your project already has components, click **Create** and then click **Create** again in the REST API card. If not, click **Create** in the  REST API card.
+
+3. Enter a unique name and a description for the API. For example, you can enter the name and the description given below:
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Name**        | `readingList`           |
+    | **Description** | `Manages reading lists` |
+
+4. In the **Access Mode** field, leave the default selection unchanged so that users outside your organization can access your API.
+
+5. Click **Next**.
+
+6. Click **Authorize with GitHub**.
+
+7. Enter your GitHub credentials, and select the private repository you created by forking  [https://github.com/Asitha/choreo-reading-list-application-example](https://github.com/Asitha/choreo-reading-list-application-example) to install the [Choreo GitHub App](https://github.com/marketplace/choreo-apps).
+
+    !!! info
+         The **Choreo GitHub App** requires the following permission:<br/><br/>- Read and write access to code and pull requests.<br/><br/>- Read access to issues and metadata.<br/><br/>You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if you do not want Choreo to have access to your GitHub account. However, write access is only used to send pull requests to a user repository. Choreo will not directly push any changes to a repository.
+
+     ![Authorize GitHub app](../assets/img/tutorials/connect-own-repo/authorize-github-app.png){.cInlineImage-half}
+
+
+8. In the **Connect Repository** dialog box, enter the following information:
+
+    | **Field**             | **Description**                               |
+    |-----------------------|-----------------------------------------------|
+    | **GitHub Account**    | Your account                                  |
+    | **GitHub Repository** | `**choreo-reading-list-application-example**` |
+    | **Branch**            | `**version-1**`                               |
+    | **Path**              | `**reading-list-service**`                    |
+
+9. Click **Create** to proceed with component initialization. This displays a pull request similar to the following:
```

Comment:
component initialization?

Reply:
Updated this sentence to make it clearer what happens here.